internal/database/migrations/migrations.go

```diff
@@ -0,0 +1,361 @@
+package migrations
```

Comment:
There is a lot of code here, but it's unclear what its actual purpose is...

Reply:
mostly copied over from kafka fleet manager..  it there in case we want the CLI to finer control over rolling forward and back.  It also has some helpers to make it easier to implement migrations.

Reply:
for example: you can use 
```go
type Person struct {
	Age int
}
AddTableColumnsAction(&Person{})
```
to handle creating the Migrate and Rollback that adds/removes the `age` col.